src/native/libs/System.Native/pal_io.c

```diff
@@ -61,12 +61,32 @@ extern int     getpeereid(int, uid_t *__restrict__, gid_t *__restrict__);
 #endif
 #endif
 
-// The portable build is performed on RHEL7 which doesn't define FICLONE yet.
-// Ensure FICLONE is defined for all Linux builds.
 #ifdef __linux__
```

Comment:
nit: this can be
```suggestion
#ifdef TARGET_LINUX
```

Reply:
Is it preferable? Does it mean the same thing? What happens for Android?

Reply:
We have all the targets defined in `eng/native/configurecompiler.cmake`. We try to use them as much as possible. In a few cases (like src/native/minipal), we can't, so we use system defined macros.

> What happens for Android

`TARGET_LINUX` includes Android. To exclude Android, we use `if defined(TARGET_LINUX) && !defined(TARGET_ANDROID)` (but it is not the case here since __linux__ is defined on Android).

Reply:
I've changed it to use `TARGET_LINUX`.